Parser/Parser.php

```diff
@@ -92,4 +118,20 @@ public function parse(string $contents, string $fileName): array
         return $containerCallVisitor->getCalls();
     }
 
+    /**
+     * logErrors
+     *
+     * @param OutputInterface|null $output
+     */
+    private function logErrors(OutputInterface $output = null)
+    {
+        if ($output === null) {
+            return;
+        }
+
+        $output->writeln("");
+        foreach ($this->errors as $error) {
+            $output->writeln($error);
```

Comment:
There is a definition for a `<warn>` style - this may be a good place to use it.

Reply:
done

Reply:
:ok: